CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.14.2 (2024-01-18)
+
+Version 2.14.2 is a hotfix release for master nodes that mainly fixes
+excessive disk usage caused by the Influx DB perfdata writer feature.
+
+* Influx DB: truncate timestamps to whole seconds to save disk space. #9969
```

Comment:
I think (`InfluxDB Writers`) would fit better.

Reply:
Already considered it, but, honestly speaking, has anyone ever reported/tested the v2 writer? If no, I'd keep it like this, hiding that info behind the abstract "feature".

Reply:
> honestly speaking, has anyone ever reported/tested the v2 writer?

See https://github.com/Icinga/icinga2/pull/9962#issuecomment-1894176189